spec/controllers/approval_requests_controller_spec.rb

```diff
@@ -362,5 +369,12 @@
         expect { send_request }.not_to change(ApprovalRequest, :count)
       end
     end
+
+    it 'sends declined vacation email notification' do
+      sign_in user
+
+      expect { send_request }
```

Comment:
Does it send notification for any request? What happens in case of:
    unauthorized access;
    DB error, like there is no such a request in DB.